tests/globalkilltest/global_kill_test.go

```diff
@@ -396,125 +389,130 @@ func sleepRoutine(ctx context.Context, sleepTime int, conn *sql.Conn, connID uin
 }
 
 // NOTICE: db1 & db2 can be the same object, for getting conn1 & conn2 from the same TiDB instance.
-func (s *TestGlobalKillSuite) killByKillStatement(c *C, db1 *sql.DB, db2 *sql.DB, sleepTime int) time.Duration {
+func (s *GlobalKillSuite) killByKillStatement(t *testing.T, db1 *sql.DB, db2 *sql.DB, sleepTime int) time.Duration {
 	ctx := context.TODO()
 
 	conn1, err := db1.Conn(ctx)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	defer conn1.Close()
 
 	var connID1 uint64
 	err = conn1.QueryRowContext(ctx, "SELECT CONNECTION_ID();").Scan(&connID1)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	log.Info("connID1", zap.String("connID1", "0x"+strconv.FormatUint(connID1, 16)))
 
 	ch := make(chan sleepResult)
 	go sleepRoutine(ctx, sleepTime, conn1, connID1, ch)
 
 	time.Sleep(waitToStartup) // wait go-routine to start.
 	conn2, err := db2.Conn(ctx)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	defer conn2.Close()
 
 	var connID2 uint64
 	err = conn2.QueryRowContext(ctx, "SELECT CONNECTION_ID();").Scan(&connID2)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	log.Info("connID2", zap.String("connID2", "0x"+strconv.FormatUint(connID2, 16)))
 
 	log.Info("exec: KILL QUERY",
 		zap.String("connID1", "0x"+strconv.FormatUint(connID1, 16)),
 		zap.String("connID2", "0x"+strconv.FormatUint(connID2, 16)),
 	)
 	_, err = conn2.ExecContext(ctx, fmt.Sprintf("KILL QUERY %v", connID1))
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 
 	r := <-ch
-	c.Assert(r.err, IsNil)
+	require.NoError(t, err)
 	return r.elapsed
 }
 
 // [Test Scenario 1] A TiDB without PD, killed by Ctrl+C, and killed by KILL.
-func (s *TestGlobalKillSuite) TestWithoutPD(c *C) {
+func TestWithoutPD(t *testing.T) {
+	s, clean := createGloabalKillSuite(t)
+	defer clean()
 	var err error
 	port := *tidbStartPort
 	tidb, err := s.startTiDBWithoutPD(port, *tidbStatusPort)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	defer s.stopService("tidb", tidb, true)
 
 	db, err := s.connectTiDB(port)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	defer func() {
 		err := db.Close()
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 	}()
 
-	const sleepTime = 2
+	sleepTime := 2
 
 	// Test mysql client CTRL-C
 	// mysql client "CTRL-C" truncate connection id to 32bits, and is ignored by TiDB.
-	elapsed := s.killByCtrlC(c, port, sleepTime)
-	c.Assert(elapsed, GreaterEqual, sleepTime*time.Second)
+	elapsed := s.killByCtrlC(t, port, sleepTime)
+	log.Info("elapsed", zap.Any("elapsed", elapsed))
+	require.GreaterOrEqual(t, elapsed.Seconds(), float64(sleepTime))
 
 	// Test KILL statement
-	elapsed = s.killByKillStatement(c, db, db, sleepTime)
-	c.Assert(elapsed, Less, sleepTime*time.Second)
+	elapsed = s.killByKillStatement(t, db, db, sleepTime)
+	require.Less(t, elapsed.Seconds(), float64(sleepTime))
```

Comment:
Why not `require.Less(t, elapsed, sleepTime*time.Second)`? Any reason for the change?